go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/schollz/hostyoself
 go 1.12
 
 require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
```

Comment:
This was added after running `go mod tidy` and is because of `urfave/cli` if I recall correctly.